providers/provider.go

```diff
@@ -0,0 +1,15 @@
+package providers
+
+import (
+	"context"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+// IsProvider is the interface all providers need to implement
+type IsProvider func(ctx context.Context, k8sClient kubernetes.Interface) (bool, error)
```

Comment:
With the interface changes this is not used anymore

Reply:
fixed